Sources/AccessibilitySnapshot/Core/Swift/Classes/AccessibilityHierarchyParser.swift

```diff
@@ -741,6 +747,31 @@ fileprivate extension NSObject {
         }
         return []
     }
+
+    var identifier: String? {
+        if let accessibilityIdentifier = value(forKey: "accessibilityIdentifier") as? String {
+            return accessibilityIdentifier
+        }
+        return nil
+    }
```

Comment:
There is no "clean" way to access the `accessibilityIdentifier` for an `NSObject`, since it's only available for objects that conform to `UIAccessibilityIdentification` ([doc](https://developer.apple.com/documentation/uikit/uiaccessibilityidentification)) I tried using the `accessibilityIdentifierBlock`, but that yielded no results, so I've opted to pull the value using key values. I'm open to suggestions though if ya'll have any!

Reply:
I tried this, but it was not triggering any of the UIKIt views.

Reply:
Ugh, okay yes I vaguely remember now having a discussion with @RoyalPineapple about something very similar. I forget the context, but perhaps he can recall what he ended up doing here.

Reply:
I think the solution here of testing for known conformance before using the key/value as a fallback works well enough. This bug is frustrating every time it comes up.